stable/kube-bench/templates/cron.yaml

```diff
@@ -1,12 +1,14 @@
+---
 apiVersion: {{ template "cronjob.apiVersion" . }}
 kind: CronJob
 metadata:
-  name: kube-bench
+  name: {{ .Values.jobName }}
```

Comment:
It's more conventional to use a templated value here:

```suggestion
  name: {{ include "test.fullname" . }}
```

You would needs to update `stable/kube-bench/templates/_helpers.tpl` also with the function.

You can copy the output from `helm create example-chart` and see how it should be done. Would that be OK?

Reply:
good point, added
i also added the labels from the example chart
![image](https://user-images.githubusercontent.com/874007/225557191-88829baa-1df1-4602-bb7f-a6ec95688201.png)